pkg/volume/rbd/rbd_util.go

```diff
@@ -268,8 +281,9 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) error {
 
 	devicePath, found := waitForPath(b.Pool, b.Image, 1)
 	if !found {
-		// modprobe
-		_, err = b.plugin.execCommand("modprobe", []string{"rbd"})
+		// modprobe on the host
+		exe := exec.New()
```

Comment:
Fixed both in this PR and in the proposal: https://github.com/kubernetes/community/pull/589/files#diff-c1f7e2debf21346978bae50b5f7e37d2R70

(not sure how stable is the link)